lib/Field/FieldContent.js

```diff
@@ -14,13 +21,21 @@ export function FieldContent({ children, className, instructions }) {
         className,
         'col-span-12 md:col-span-8',
         'col-start-1',
-        'border border-solid border-neutral-90',
+        'border-solid',
         'rounded',
         'shadow',
+        'transition-all, transition-ease duration-200',
         {
           'p-6': !inStructureEditMode,
           'md:col-start-5': !inStructureEditMode,
-          'md:col-start-4': inStructureEditMode
+          'md:col-start-4': inStructureEditMode,
+          'border border-neutral-90': status === ACTIVE,
+          'border border-neutral-90 bg-neutral-98 group-hover:bg-white':
```

Comment:
Nice use of group-hover 👍 I guess the group class is already set on a parent somewhere?